src/WebJobs.Extensions.DurableTask/DurableOrchestrationContext.cs

```diff
@@ -35,13 +36,15 @@ public sealed class DurableOrchestrationContext : DurableOrchestrationContextBas
         private string serializedCustomStatus;
         private int owningThreadId;
 
+        private List<Func<Task>> deferedFunctions;

```

Comment:
Please make this `readonly`. Also, I suggest naming this `deferredTasks` or `deferredActions` (note the spelling) instead of `deferedFunctions` because people might confuse "functions" with "Azure Functions".